crates/sui-protocol-config/src/lib.rs

```diff
@@ -2807,6 +2819,15 @@ impl ProtocolConfig {
                         // Enable Mysticeti fastpath for devnet
                         cfg.feature_flags.mysticeti_fastpath = true;
                     }
+
+                    if chain != Chain::Mainnet && chain != Chain::Testnet {
+                        cfg.feature_flags.uncompressed_g1_group_elements = true;
+
+                        cfg.group_ops_bls12381_g1_to_uncompressed_cost = Some(26);
+                        cfg.group_ops_bls12381_g1_from_uncompressed_cost = Some(52);
+                        cfg.group_ops_bls12381_g1_sum_of_uncompressed_base_cost = Some(26);
+                        cfg.group_ops_bls12381_g1_sum_of_uncompressed_cost_per_term = Some(13);
```

Comment:
let's move those outside the if (as we can change them globally) and leave in only the feature flag